pdctl/command/config_command.go

```diff
@@ -115,3 +149,50 @@ func setConfigCommandFunc(cmd *cobra.Command, args []string) {
 		printResponseError(r)
 	}
 }
+
+func setReplicateConfigCommandFunc(cmd *cobra.Command, args []string) {
```

Comment:
can we use `setConfigCommandFunc` directly? I think the implementation code is duplicated here.